File: C/section06/S06E04.c
#include <stdio.h>
#include <ctype.h>

int main() {
    float height;
    float result;
    char gender;

    printf("Please type a height: \n");
    scanf("%f", &height);

    gets(stdin); // input bug fix

    printf("Please type a gender f or m: \n");
    scanf("%c", &gender);

    if (tolower(gender) == 'm') {
        result = (72.7 * height) - 58;
        printf("\nThe ideal BMI for this height and gender is %.2f.", result);
    } else if (tolower(gender) == 'f') {
        result = (62.1 * height) - 44.7;
        printf("\nThe ideal BMI for this height and gender is %.2f.", result);
    } else {
        printf ("\ngender not found.");
    }

    return 0;
}

File: Python/section06/S06e04.py
personHeight = float(input ("How tall are you in meters? "))
personGender = input("Please enter your gender f/m. ")

if personGender.lower() == 'm':
    idealBMI = (72.7 * personHeight - 58)
elif personGender.lower() == 'f':
    idealBMI = (62.1 * personHeight) - 44.7
else:
    idealBMI = 0
    print("\nGender not found.")

print("\nYour ideal BMI is: {0:.2f}".format(idealBMI))